AEPMessaging/Sources/Messaging.swift

```diff
@@ -25,32 +24,42 @@ public class Messaging: NSObject, Extension {
     public var metadata: [String: String]?
     public var runtime: ExtensionRuntime
 
-    private var messagesRequestEventId: String?
-    private var lastProcessedRequestEventId: String?
+    private var messagesRequestEventId: String = ""
+    private var lastProcessedRequestEventId: String = ""
```

Comment:
why did we change these to non-optional?  seems preferable to treating an empty string as nil

Reply:
I changed this mainly to avoid clutter because once initialized these will always contain valid values. However, if you specify as optional, you have to unwrap during every usage (even for `requestedSurfacesforEventId` dict lookup).